addons/revolutionaries/CfgVehicles_O_Tanoan.hpp

```diff
@@ -145,6 +145,103 @@ class TACU_Revolutionaries_U_O_Tanoan_Autorifleman: TACU_Revolutionaries_U_O_Tan
     editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
 };
 
+class TACU_Revolutionaries_U_O_Tanoan_FigureHeadUA: TACU_Revolutionaries_U_O_Tanoan_Rifleman01 {
+    displayName = "Figure Head (Unarmed)";
+    uniformClass = "U_I_C_Soldier_Para_2_F";
+    linkedItems[] = {DEFAULT_ITEMS, "H_Beret_blk"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "", "H_Beret_blk"};
+    backpack = "";
+    weapons[] = {"Throw", "Put"};
+    respawnWeapons[] = {"Throw", "Put"};
+    magazines[] = {""};
+    respawnMagazines[] = {""};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Tanoan_FigureHead: TACU_Revolutionaries_U_O_Tanoan_FigureHeadUA {
+    displayName = "Figure Head";
+    weapons[] = {"CUP_hgun_TaurusTracker455_gold", "Throw", "Put"};
+    respawnWeapons[] = {"CUP_hgun_TaurusTracker455_gold", "Throw", "Put"};
+    magazines[] = {mag_6("CUP_6Rnd_45ACP_M")};
+    respawnMagazines[] = {mag_6("CUP_6Rnd_45ACP_M")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Tanoan_Leader: TACU_Revolutionaries_U_O_Tanoan_Rifleman01 {
+    displayName = "Leader";
+    uniformClass = "U_I_C_Soldier_Para_4_F";
+    linkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "H_Booniehat_oil"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "H_Booniehat_oil"};
+    weapons[] = {"TACU_Revolutionaries_W_CZ805_GL", "Throw", "Put"};
+    respawnWeapons[] = {"TACU_Revolutionaries_W_CZ805_GL", "Throw", "Put"};
+    magazines[] = {mag_6("CUP_30Rnd_556x45_G36"), mag_6("1Rnd_HE_Grenade_shell")};
+    respawnMagazines[] = {mag_6("CUP_30Rnd_556x45_G36"), mag_6("1Rnd_HE_Grenade_shell")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
```

Comment:
```suggestion
    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
    headgearList[] = {
        "H_Booniehat_oil", 1
    };
```

CBA_facewearList as well if you don't want balaclava